appveyor.yml

```diff
@@ -53,6 +53,7 @@ install:
     - cmd: conda config --add channels http://conda.binstar.org/conda-forge
     - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```

Comment:
I actually just notice that. I was confused due to the same version we had to roll bacl.

> We need to add this in a new PR to conda-smithy with @melund's change.

Not sure about that. I am fixing the new feedstocks only and this is fixed in staged recipes. So let's hope that a new conda-build is out soon to avoid modifying `conda-smithy`.